lib/cogctl/actions/bundles/create.ex

```diff
@@ -27,12 +29,16 @@ defmodule Cogctl.Actions.Bundles.Create do
 
   def option_spec do
     [{:file, :undefined, :undefined, {:string, :undefined}, 'Path to your bundle config file (required)'},
-     {:templates, :undefined, 'templates', {:string, 'templates'}, 'Path to your template directory'}]
+     {:templates, :undefined, 'templates', {:string, 'templates'}, 'Path to your template directory'},
+     {:enabled, :undefined, 'enable', {:boolean, false}, 'Enable bundle after installing'},
+     {:"relay-groups", :undefined, 'relay-groups', {:string, :undefined}, 'List of relay group names separated by commas to assign the bundle'}]
   end
 
   def run(options, _args, _config, endpoint) do
     case convert_to_params(options, [file: :required,
-                                     templates: :required]) do
+                                     templates: :required,
+                                     enabled: :optional,
+                                     "relay-groups": :optional]) do
       {:ok, params} ->
         with_authentication(endpoint, &do_create(&1, params))
       :error ->
```

Comment:
We may have a conflict here. https://github.com/operable/cogctl/pull/49

`convert_to_params` returns an error with a list missing required args now.
`{:error, {:missing_params, <missing_params>}}`